dts/bindings/i2c/nuvoton,npcx-i2c-ctrl.yaml

```diff
@@ -1,20 +1,22 @@
 # Copyright (c) 2020 Nuvoton Technology Corporation.
 # SPDX-License-Identifier: Apache-2.0
 
-description: Nuvoton, NPCX-I2C node
+description: Nuvoton, NPCX-I2C controller node
 
-compatible: "nuvoton,npcx-i2c"
+compatible: "nuvoton,npcx-i2c-ctrl"
 
 include: [i2c-controller.yaml]
 
+bus: i2c
+
 properties:
     reg:
         required: true
     clocks:
         required: true
     label:
         required: true
-    pinctrl:
-        type: phandles
+    index:
```

Comment:
As far as I can tell `index` is only used for logging error.  This should be removed from the binding and use `label` or `reg` to identify which controller is logging an error.

Reply:
It's very useful to show which i2c port and controller encountered problems. `reg` might not be straightforward enough to let hardware engineer know which pad to measure the signal. Using `label` is fine, but I need to add "char[]" array in config structure in case we need to print "label" without DT_XX macro. It costs more memory. The index property only exists during build time and cost one byte in the driver. Might I save it in case we need to print messages for debugging?

Reply:
No, the intent of the devicetree is for describing the hardware and we've avoid such properties in the past.  Such debug should be handled in the driver itself.

Reply:
Okay. I will remove "index" property.

Reply:
Done.